pom.xml

```diff
@@ -136,6 +136,11 @@
             <artifactId>commons-lang3</artifactId>
             <version>3.5</version>
         </dependency>
+        <dependency>
```

Comment:
should this been added to azure-commons too?

Reply:
This dependency is introduced by azure sdk which managed by this plugin self. It auto upgrades to new version and cause some conflicts. So I manually downgrade it. No necessary to change commons.

Reply:
During transition with Menghua, he mentioned some defect that different azure sdks will cause problem on not using right version. So dependency management is centralized in azure-commons, other plugins just call then depends on azure-commons. From what you described, seems this is similar issue.